tests/test_perf_tracking.py

```diff
@@ -642,10 +643,12 @@ def test_long_position_receives_dividend(self):
         self.assertEqual(len(results), 6)
         cumulative_returns = \
             [event['cumulative_perf']['returns'] for event in results]
-        self.assertEqual(cumulative_returns, [0.0, 0.0, 0.1, 0.1, 0.1, 0.1])
+        assert_array_almost_equal(cumulative_returns,
```

Comment:
Due to floating point errors from some extra math we need to do. In this case, the returns used to be calculated as `1.0/10.0 = 0.1`. Now it's `(1 + 0.0) * (1 + 0.1) - 1 = 0.10000000000000009`